addons/mail/wizard/mail_compose_message.py

```diff
@@ -515,7 +515,7 @@ def _process_state(self, mail_values_dict):
     def _get_blacklist_record_ids(self, mail_values_dict, recipients_info):
         blacklisted_rec_ids = set()
         if self.composition_mode == 'mass_mail':
-            self.env['mail.blacklist'].flush(['email'])
+            self.env['mail.blacklist'].flush_model(['email', 'active'])
```

Comment:
shouldn't be this in stable ?

Reply:
Actually, it something in stable (not in 15.0 but between 15.0 and master), it is just that the fix has changed it and this is just to restore it. Sorry about that.